docs/README.md

```diff
@@ -0,0 +1,115 @@
+
+## CleanFlight Configuration
+Your FC should now be running CF, and you should be able to connect to it using the Configurator. If that is not the case, please go back to the previous sections and follow the steps carefully.
+
+<!--- This next paragraph should probably contain less info, as this info already exists in Configuration.md -->
+Now, there are two ways to [configure CF](https://github.com/cleanflight/cleanflight/blob/master/docs/Configuration.md); via  the Configurator's tabs (in a "graphical" way, clicking through and selecting/changing values and tickboxes) and using the [Command Line Interface (CLI)](https://github.com/cleanflight/cleanflight/blob/master/docs/Cli.md). Some settings may only be configurable using the CLI.
+
+* It is now a good time to setup your RC Receiver and Transmitter. Set the Tx so that it outputs at least 4 channels (Aileron, Elevator, Throttle, Rudder) but preferably more. E.g. you can set channels 5 and 6 to be controlled by 3-position switches, to be used later. Maybe set up EXPO on AIL/ELE/RUD, but you should know that it can also be done in CF's software later. If using RSSI over PPM or PWM, it's now time to configure your Rx to output it on a spare channel.
+
+* Connect the Rx to the FC, and the FC to the PC. You may need to power the Rx through a BEC (its 5V rail - observe polarity!).
+
+* On your PC, connect to the Configurator, and go to the first tab. Check that the board animation is moving properly when you move the actual board. Do an accelerometer calibration.
+
+* Configuration tab: Select your aircraft configuration (e.g. Quad X), and go through each option in the tab to check if relevant for you. 
+
+ * E.g. you may want to enable ONESHOT125 for Oneshot-capable ESCs.
+ * You may need RX_PPM if you're using an RC Receiver with PPM output etc.
+ * If planning to use the battery measurement feature of the FC, check VBAT under Battery Voltage.
+ * If using analog RSSI, enable that under RSSI. Do not enable this setting if using RSSI injected into the PPM stream.
+ * Motors will spin by default when the FC is armed. If you don't like this, enable MOTOR_STOP.
+ * Also, adjust the minimum, middle and maximum throttle according to [this post](http://www.rcgroups.com/forums/showthread.php?p=29763892#post29763892).
+ * Finally, click Save and Reboot.
+
+* Receiver tab: Check that the channel inputs move according to your Tx inputs. Check that the Channel map is correct, along with the RSSI Channel, if you use that. You can also set EXPO here instead of your Tx. Click Save!
+* Modes tab: Setup the desired modes. See [this](https://github.com/cleanflight/cleanflight/blob/master/docs/Modes.md) for what each mode does, but for the beginning you mainly need HORIZON, if any.
+
+* Before finishing this section, you should calibrate the ESCs, install the FC to the frame, and connect the RSSI cable, buzzer and battery if you have chosen to use those.
+
```

Comment:
I mean the serial ports 1-4. Probably needs a pointer to the serial docs along with the notes about what sort of devices require serial ports (serial Rx, GPS, blackbox, OSD, telemetry), otherwise there's no motivation in the readme to check out the ports config.